### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by libklein** [davatorium/rofi#1487] [General]
Hi,

i was wondering whether rofi will "participate" in this year's hacktoberfest (https://hacktoberfest.digitalocean.com/). Essentially, this simply entails assigning the "hacktoberfest-accepted" label to eligible (i.e. "high quality") PRs, see https://hacktoberfest.digitalocean.com/faq.

**Comment by DaveDavenport** [comment]
No idea what hacktoberfest is.

**Reply by libklein** [reply]
So i assume that that won't be the case then?

**Reply by DaveDavenport** [reply]
no idea, I don't know what it is.
